src/tilingutils.jl

```diff
@@ -199,8 +195,248 @@ function get_tiles(array, side_length::Int)
+    w = ImageSegmentation.watershed(seg, cc)
+    lmap = labels_map(w)
+    return Images.isboundary(lmap)
+end
+
+function imextendedmin(img, h)
+    return imregionalmin(ImageSegmentation.hmin_transform(img, h))
+end
+
+function imregionalmin(A, conn=2)
+    return ImageMorphology.local_minima(A; connectivity=conn) .> 0
+end
+
+function impose_minima(I::AbstractArray{T}, BW::AbstractArray{Bool}) where {T<:Integer}
+    marker = 255 .* BW
+    mask = imcomplement(min.(I .+ 1, 255 .- marker))
+    reconstructed = IceFloeTracker.MorphSE.mreconstruct(
+        IceFloeTracker.MorphSE.dilate, marker, mask
+    )
+    return IceFloeTracker.imcomplement(Int.(reconstructed))
+end
+
+function impose_minima(
+    I::AbstractArray{T}, BW::AbstractMatrix{Bool}
+) where {T<:AbstractFloat}
+    # compute shift
+    b, a = extrema(I)
+    rng = b - a
+    h = rng == 0 ? 0.1 : rng / 1000
+
+    marker = -Inf * BW .+ (Inf * .!BW)
+    mask = min.(I .+ h, marker)
+
+    return 1 .- IceFloeTracker.MorphSE.mreconstruct(
+        IceFloeTracker.MorphSE.dilate, 1 .- marker, 1 .- mask
+    )
+end
+
+# TODO: Add tests for get_new2 and get_new3
+"""
+    getnew2(morph_residue, local_maxima_mask, factor, segment_mask, L0mask)
+
+
+Calculate the new image `new2` from the input image `morph_residue`.
+
+# Arguments
+- `morph_residue`: The morphological residue image.
+- `local_maxima_mask`: The local maxima mask.
+- `factor`: The factor to apply to the local maxima mask.
+- `segment_mask`: The segment mask -- intersection of bw1 and bw2 in first tiled workflow of `master.m`.
+- `L0mask`: zero-labeled pixels from watershed.
+"""
+function get_new2(morph_residue, local_maxima_mask, factor, segment_mask, L0mask)
+    new2 = to_uint8(morph_residue .+ local_maxima_mask .* factor)
+    new2[segment_mask .|| L0mask] .= 0
+    return MorphSE.fill_holes(new2)
+end
+
+"""
+    get_new3(new2, L0mask, radius, amount, local_maxima_mask, factor, segment_mask)
+
+Calculate the new image `new3` from the input image `new2`.
+
+# Arguments
+- `img`: The input image.
+- `L0mask`: zero-labeled pixels from watershed.
+- `radius`: The radius of the unsharp mask.
+- `amount`: The amount of unsharp mask.
+- `local_maxima_mask`: The local maxima mask.
+- `factor`: The factor to apply to the local maxima mask.
+- `segment_mask`: The segment mask -- intersection of bw1 and bw2 in first tiled workflow of `master.m`.
+
+"""
+function get_new3(img, L0mask, radius, amount, local_maxima_mask, factor, segment_mask)
+    new3 = unsharp_mask(img, radius, amount, 255)
+    new3[L0mask] .= 0
+    new3 = reconstruct(new3, se, "dilation", false)
+    new3[segment_mask] .= 0
+    return to_uint8(new3 + local_maxima_mask .* factor)
+end
+
+"""
+    get_final(img, label, segment_mask, se_erosion, se_dilation)
+
+Final processing following the tiling workflow.
+"""
+function get_final(img, label, segment_mask, se_erosion, se_dilation)
+    img = hbreak(img)
+
+    # doesn't do much/anything
```

Comment:
I'm fine with leaving the "label==1" out entirely now that we know it doesn't do anything.